sdk/account_id.go

```diff
@@ -98,17 +98,20 @@ func AccountIDFromEvmPublicAddress(s string) (AccountID, error) {
 
 // AccountIDFromSolidityAddress constructs an AccountID from a string
 // representation of a _Solidity address
+// "000000000000000000000000000000000000138D" -> 0.0.59 ?
+// "302a300506032b6570032100114e6abc371b82da" -> 0.0.302a300506032b6570032100114e6abc371b82da ?
+// Deprecated
 func AccountIDFromSolidityAddress(s string) (AccountID, error) {
-	shard, realm, account, err := _IdFromSolidityAddress(s)
+	evmAddress, err := hex.DecodeString(s)
 	if err != nil {
 		return AccountID{}, err
 	}
-
 	return AccountID{
-		Shard:    shard,
-		Realm:    realm,
-		Account:  account,
-		checksum: nil,
+		Shard:           0,
+		Realm:           0,
+		Account:         0,
+		checksum:        nil,
+		AliasEvmAddress: &evmAddress,
 	}, nil
 }
```

Comment:
we would not change the return on this. Rather mark it as deprecated and if someone needed to create an entity ID out of the EVM address there should be either an existing method or new method created to provide such functionality.